trade-finance-manager-ui/templates/case/underwriting/amendments/amendment-banks-decision/amendment-banks-decision.njk

```diff
@@ -0,0 +1,61 @@
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% import './_macros/banks-decision-submitted.njk' as banksDecisionSubmitted %}
+
+{% macro render(params) %}
+
+  {% set banksDecision = params.amendmentBanksDecision %}
+
+  {% block underwriting_content %}
+
+    <div class="govuk-grid-row">
+
+      <div class="govuk-grid-column-three-quarters">
+
+        {% if banksDecision.amendment.amendments.underwriterManagersDecision.decision %}
+
+          {% if banksDecision.amendment.amendments.banksDecision.decision %}
+
+            {% set banksDecisionSubmittedComponentData = {
+              decision: banksDecision.amendment.amendments.banksDecision.decision,
+              user: banksDecision.amendment.amendments.banksDecision.user
+            } %}
+
+            {{banksDecisionSubmitted.render(banksDecisionSubmittedComponentData)}}
+
+          {% else %}
+
+            {% if banksDecision.userCanEdit %}
+              <div class="govuk-!-padding-top-0">
+                {{ govukButton({
+                    text: "Add decision",
+                    href: "/case/" + banksDecision.amendment.amendments.dealId + "/facility/" + banksDecision.amendment.amendments.facilityId + "/amendment/" + banksDecision.amendment.amendments.amendmentId + "/banks-decision",
```

Comment:
same as the other one, can we simplify this objects?

Reply:
done